### PR TITLE
use gulp-connect over express

### LINKED ISSUE #8: Use gulp-connect?
https://www.npmjs.com/package/gulp-connect

Any reason we can't use this like in https://github.com/touchstonejs/touchstonejs ?

### PR DESCRIPTION
**Pull request by dcousens**
Resolves #8

### CODE REVIEW

**Comment by dcousens** [comment]
This also means [naturally] that live reloads now work.

File: index.js [hunk 1]
```diff
@@ -13,19 +14,23 @@ var watchify = require('watchify');
 var xtend = require('xtend');
 
 module.exports = function (gulp) {
-	function bundler (target, name, dest) {
+	function doBundle (target, name, dest) {
```
Comment (dcousens) [inline index.js:17]: While I was figuring out this code,  I changed this.  Just putting it back to what it was, it's a good name anyway.

**Comment by dcousens** [comment]
@JedWatson please release a new version if you merge.

**Comment by JedWatson** [comment]
done

**Comment by JedWatson** [comment]
@dcousens 0.2.2 up

**Pull request merged**